tasks/assert.yml

```diff
@@ -0,0 +1,92 @@
+---
+- name: test if rsyslog_receiver is set correctly
```

Comment:
Not quite.

Reply:
Delete in the last version.

Reply:
That's should be done on the last version.